docs/source/concepts/ledger-model/ledger-privacy.rst

```diff
@@ -162,6 +170,28 @@ conformant. Lastly, as projections carry no information about the
 requesters, we cannot talk about authorization on the level of
 projections.
 
+
```

Comment:
This section was already quite long previously; since the new content is not tightly integrated to the old, I'd split it off into a different section ("Privacy Through Authorization" or somesuch)